### PR TITLE
Add basic latency alerts as Terraform config

### PR DESCRIPTION
**Pull request by yegle**
#1036

### CODE REVIEW

File: terraform/alerting/alerts.tf [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  slow_services = format("(%s)", join("|", [
+    "export",
```
Comment (mikehelmick) [inline terraform/alerting/alerts.tf:17]: also needs cleanup services

Comment (yegle) [reply terraform/alerting/alerts.tf:17]: export service actually rarely exceeds 10s at 99 percentile.

It looks like based on the last 1 month of data, only cleanup-exposure and cleanup-export should have 20s of alerting threshold.

image

File: terraform/alerting/alerts.tf [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  slow_services = format("(%s)", join("|", sort([
+    "cleanup-export",
```
Comment (mikehelmick) [inline terraform/alerting/alerts.tf:17]: export not being slow is an artifact of our setup (we may need to add it back)

Comment (yegle) [reply terraform/alerting/alerts.tf:17]: I see. I added it to the SLOWEST tier instead.

**Review by mikehelmick — APPROVED** [review]

**Pull request merged**